### PR TITLE
Add multi language support in User schema

### PR DESCRIPTION
**Pull request by m4v15**
relates #21

### CODE REVIEW

**Comment by m4v15** [comment]
@des-des can you take a look at this

**Review by des-des — APPROVED** [review]

File: tests/index.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ dbConnection.once('open', () => {
   require('./controllers/place.test.js')
   require('./controllers/event.test.js')
   require('./controllers/product.test.js')
-  
+
```
Comment (mattlub) [inline tests/index.test.js:10]: 👍

File: src/models/User.js [hunk 2]
```diff
@@ -1,22 +1,32 @@
 const mongoose = require('mongoose')
 
 const { roles } = require('./constants.json')
+const { customRequireValidator } = require('../db/utils')
+
+const UserTranslatedFieldsSchema = mongoose.Schema(
```
Comment (mattlub) [inline src/models/User.js]: camelCase habibi

**Review by mattlub — CHANGES_REQUESTED** [review]

**Review by mattlub — APPROVED** [review]

File: tests/index.test.js (hunk 1 above)
Comment (des-des) [reply tests/index.test.js:10]: :tada:

File: src/db/utils.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const customRequireValidator = function (next) {
```
Comment (des-des) [inline src/db/utils.js:1]: Not sure about this name.

`lanEntryExistsValidator` (thats also awful but maybe better?)

**Review by des-des — APPROVED** [review]

**Pull request merged**